web/styles/message_edit_history.css

```diff
@@ -39,6 +39,10 @@
 
     .messagebox-content {
         display: block !important;
+
+        .message_content {
+            cursor: default !important;
+        }
```

Comment:
Why is this the right selector? `cursor: pointer` is on `.messagebox` in `message_row.css`...

Reply:
`cursor: pointer`, for .messagebox is being overwritten by `cursor: auto` by `app_components.css`. 

![image](https://github.com/zulip/zulip/assets/100200105/daa6cab5-cde2-4530-b67c-c34e470ac4de)

on hovering over `.messagebox .message_content` the `cursor` is set to pointer from `app_components.css`

![on hover message_content](https://github.com/zulip/zulip/assets/100200105/c3b504e8-d48e-4e6e-aef5-8211491e0591)

Reply:
That's why I have used `.message_content` as CSS selector.
![image](https://github.com/zulip/zulip/assets/100200105/beea6c2d-91ed-4a5d-96fe-aed0fe0370f5)